src/mvPlotting.cpp

```diff
@@ -1327,8 +1303,14 @@ DearPyGui::draw_2dhistogram_series(ImDrawList* drawlist, mvAppItem& item, const
 		xptr = &(*config.value.get())[0];
 		yptr = &(*config.value.get())[1];
 
-		ImPlot::PlotHistogram2D(item.info.internalLabel.c_str(), xptr->data(), yptr->data(), (int)xptr->size(),
-			config.xbins, config.ybins, config.density, ImPlotLimits(config.xmin, config.xmax, config.ymin, config.ymax), config.outliers);
+		ImPlotRect range = ImPlotRect();
```

Comment:
`range`: this variable is unused, probably a leftover of some development efforts. We can safely remove the lines 1306-1310.